### ISSUE TITLE
When will the AUR package get fixed?

### ISSUE THREAD

**Issue by raaron773** [Stremio/stremio-shell#349]
Both Stremio and Stremio-Beta are not working.
And both fail at the same error:
```
Install the project...
-- Install configuration: "Release"
-- Installing: /home/$USER/stremio-beta/pkg/stremio-beta/opt/stremio/stremio
make[1]: Leaving directory '/home/$USER/stremio-beta/src/stremio/build'
install -Dm 644 server.js "/home/$USER/stremio-beta/pkg/stremio-beta/opt/stremio/server.js"
install -Dm 644 smartcode-stremio.desktop "/home/$USER/stremio-beta/pkg/stremio-beta/opt/stremio/smartcode-stremio.desktop"
cp -r icons "/home/$USER/stremio-beta/pkg/stremio-beta/opt/stremio/"
ln -s "" "/home/$USER/stremio-beta/pkg/stremio-beta/opt/stremio/node"
ln: failed to create symbolic link '/home/$USER/stremio-beta/pkg/stremio-beta/opt/stremio/node' -> '': No such file or directory
make: *** [release.makefile:18: install] Error 1
==> ERROR: A failure occurred in package().
    Aborting...
```
It's been broken for almost a year now. And the fix was by [dyatlov86](https://github.com/dyatlov86) which basically said to add the following: `sudo sed -i 's/details.family.toLowerCase()/details.family.toString().toLowerCase()/g' /opt/stremio/server.js`
But how will that work for users who are installing it for first time? There's no `/opt/stremio/server.js` to edit in the first place.

**Comment by core1024** [comment]
The proposed fix has nothing to do with the error you get.
These are the relevant lines in the error:
```
ln -s "" "/home/$USER/stremio-beta/pkg/stremio-beta/opt/stremio/node"
ln: failed to create symbolic link '/home/$USER/stremio-beta/pkg/stremio-beta/opt/stremio/node' -> '': No such file or directory
make: *** [release.makefile:18: install] Error 1
```
You can see that the `ln` command tries to link **nothing** and from the last line of the error message is obvious that this error occurred in [release.makefile on line 18](https://github.com/Stremio/stremio-shell/blob/306892da7515685347673e3600b79c212c16ba84/release.makefile#L18) where the `ln` command is called. For some reason `which node` command fails to find your `nodejs` installation. As `nodejs` is [dependency](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=stremio-beta#n12) for `stremio-beta` this shouldn't happen. I presume that something is wrong with your `nodejs` install. Try reinstalling or removing it before installing `stremio-beta`. You can also verify that `which node` returns a correct path to the `nodejs` binary.

**Comment by raaron773** [comment]
@core1024 OMG thank you so much! The sad part is node was installed it was the `which`  application which wasn't installed.
When I ran `which` I got `bash: which: command not found` 
I thought this gets installed by default. I tried to search online as to what the program is called but didn't found anything. So I jsut ran `sudo pacman -U which` and it worked.
Oh man I feel so stupid. Thank you for helping btw
